src/runtime/dom-particle.ts

```diff
@@ -11,6 +11,15 @@
 
 import {XenStateMixin} from '../../modalities/dom/components/xen/xen-state.js';
 import {DomParticleBase} from './dom-particle-base.js';
+import {Collection, Variable} from './handle.js';
+
+interface StatefulDomParticle extends DomParticleBase {
+  // types go here?
+}
```

Comment:
Doesn't seem to be any point in having this intermediate interface; it isn't used anywhere.

Reply:
Agreed. I was attempting to bind this to `XenStateMixin(DomParticleBase)` somehow, but I never achieved it.